CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Changelog
 
```

Comment:
@sunil-lakshman Add latest version that we going to release in changelog with live date